.github/workflows/example_0.10.yml

```diff
@@ -0,0 +1,79 @@
+name: examples_0.10
+
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'info'
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install apt-get packages
+      run: |
+        sudo ACCEPT_EULA=Y apt-get update
+        sudo ACCEPT_EULA=Y apt-get upgrade
+        sudo apt-get install wget git curl software-properties-common build-essential
+    - name: Install Rust target
+      run: |
+        rustup target add wasm32-wasi
+    - name: Install WasmEdge
+      run: |
+        VERSION=0.10.0-alpha.1
```

Comment:
If you want to remove the wasmedge_process error message, updating to `0.10.0-alpha.2` or `0.10.0-rc.1 `can help.